osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/infra/jms/messageprocessor/DomainResponseMessageProcessor.java

```diff
@@ -82,19 +84,19 @@ public void init() {
     }
 
     @Override
-    public void processMessage(final ObjectMessage message) throws JMSException {
+    public void processMessage(final ObjectMessage message) {
         LOGGER.debug("Processing smart metering response message");
 
         String correlationUid = null;
         String messageType = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "messageType" which hides the field declared at line 64. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AHiddenFieldCheck)